trade-finance-manager-ui/templates/case/tasks/_macros/tasks-table.component-test.js

```diff
@@ -63,20 +63,21 @@ describe(component, () => {
   });
 
   describe('table headers', () => {
-    it('should render `assigned to` header', () => {
-      wrapper.expectText('[data-cy="tasks-table-header-assigned-to"]').toRead('Assigned to');
-    });
-
-    it('should render `team` header', () => {
-      wrapper.expectText('[data-cy="tasks-table-header-team"]').toRead('Team');
-    });
-
-    it('should render `date started` header', () => {
-      wrapper.expectText('[data-cy="tasks-table-header-date-started"]').toRead('Date started');
-    });
+    describe.each([
+      ['Task', 'tasks-table-header-task'],
+      ['Assigned to', 'tasks-table-header-assigned-to'],
+      ['Team', 'tasks-table-header-team'],
+      ['Date started', 'tasks-table-header-date-started'],
+      ['Date completed', 'tasks-table-header-date-completed'],
+      ['Status', 'tasks-table-header-status'],
+    ])('`%s` header', (headerText, dataCy) => {
```

Comment:
What do these `describe` blocks end up saying in the output? Is it e.g. `Task tasks-table-header-task header` and is that what you were aiming for?